expertise/create_dataset.py

```diff
@@ -36,12 +36,31 @@ def convert_to_list(self, config_invitations):
         assert isinstance(invitations, list), 'Input should be a str or a list'
         return invitations
 
+    def get_paper_notes(self, author_id, dataset_params):
+
+        use_bids_as_expertise = dataset_params.get('bid_as_expertise', False)
+
+        if use_bids_as_expertise:
+            bid_invitation = dataset_params['bid_invitation']
+            bids = openreview.tools.iterget_edges(self.openreview_client, invitation=bid_invitation, tail=author_id)
```

Comment:
Maybe we should check if this value exists in the `dataset_params`. Otherwise, it will fail

Reply:
I would like this to fail. If there is not bid invitation then no bid dataset can be created.